services/parsers/symbolscraper/model/prediction.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+
+from pydantic import BaseModel, Field, conint, conlist, validator
+
+
+_SPAN_BOX_LEN = 5
+
+
+class Span(BaseModel):
```

Comment:
should `Span` and `SpanGroup` be imported from the library instead of defined here?